docs/resource-policy/policy/balloons.md

```diff
@@ -154,6 +154,9 @@ Balloons policy parameters:
     separate `cpu.classes` objects, see below.
   - `preferCloseToDevices`: prefer creating new balloons close to
     listed devices. List of strings
+  - `coreType`:  specifies preferences of the core type which could
```

Comment:
As this option is handled as a preference, that is, inability to honor the configuration choice is not an error, would you name it `preferCoreType`?